src/Servers/Kestrel/Kestrel/src/WebHostBuilderKestrelExtensions.cs

```diff
@@ -29,6 +29,10 @@ public static class WebHostBuilderKestrelExtensions
         /// </returns>
         public static IWebHostBuilder UseKestrel(this IWebHostBuilder hostBuilder)
         {
+#pragma warning disable CA1416 // Validate platform compatibility
```

Comment:
What does this mean?

Reply:
It's because of https://github.com/dotnet/aspnetcore/blob/85180121312ec21d6f33d99acb2723868586a3a2/src/Servers/Kestrel/Transport.Quic/src/AssemblyInfo.cs#L9-L11

Should they be removed? We don't have those attributes anywhere else. I'm guessing they were added because System.Net.Quic has those attributes. A NoWarn would do the same thing.

Reply:
```
D:\github\AspNetCore\src\Servers\Kestrel\Kestrel\src\WebHostBuilderKestrelExtensions.cs(32,13): error CA1416: This call site is reachable on all platforms. 'WebHostBuilderQuicExtensions.UseQuic(IWebHostBuilder)' is only supported on: 'linux', 'macOS/OSX', 'windows'. [D:\github\AspNetCore\src\Servers\Kestrel\Kestrel\src\Microsoft.AspNetCore.Server.Kestrel.csproj]
```
I've changed it to a nowarn on the quic transport assembly.